custom_components/tuya_local/devices/thermoval_tvt40_thermostat.yaml

```diff
@@ -0,0 +1,315 @@
+name: Thermoval WiFi Thermostat
+products:
+  - id: jIRwsT5yLPJO41Po
+    name: TVT 40
+primary_entity:
+  entity: climate
+  translation_key: thermostat
+  dps:
+    - id: 2
+      name: temperature
+      type: integer
+      range:
+        min: 50
+        max: 400
+      mapping:
+        - scale: 10
+          step: 5
+    - id: 3
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: Smart
+          value: program
+          icon: "mdi:home-thermometer"
+        - dps_val: Anti_frozen
+          value: away
+          icon: "mdi:snowflake-melt"
+        - dps_val: Manual
+          value: manual
+          icon: "mdi:cursor-pointer"
+    - id: 4
+      name: fault
+      type: integer
+    - id: 5
+      name: room_temperature
+      type: integer
+      mapping:
+        - scale: 10
+    - id: 6
+      name: floor_temp_limit
+      type: integer
+      mapping:
+        - scale: 10
+    - id: 7

```

Comment:
Attributes that are not used in the climate platform, and have separate entities under secondary_entities should not be listed here.  Basically the attributes listed under climate should fall under one of the following:
1. standard attributes of the climate platform
2. attributes that are used in conditions to create advanced behavior
3. read-only attributes that do not make sense as separate entities on their own